### DISCUSSION TITLE
JPA/Hibernate con SQL server

### DISCUSSION THREAD

**Discussion by 23737k** [dds-jv/foro#309] [Consultas de herramientas] [answered]
Hola, cuando voy a agregar la dependencia de 
`<dependency>
   		 <groupId>mysql</groupId>
   		 <artifactId>mysql-connector-java</artifactId>
   		 <version>X.X.X</version> 
        	</dependency>`

No sé que poner si tengo sql server. Busqué y encontré esto
`< dependency>
   		 <groupId>com.microsoft.sqlserver</groupId>
   		 <artifactId>mmssql-jdbc</artifactId>
   		 <version>8.1.1.jre8</version> 
        	</dependency>`
, pero me tira un error de missing artifact.

**Comment by gastonprieto** [comment, marked as answer]
Buenas @23737k (seria mejor si supiera tu nombre jajaa)

Fijate si con lo que dice este post de stackoverflow te sirve: https://stackoverflow.com/questions/3585544/how-to-configure-hibernate-config-file-for-sql-server

Lo que dice que hay que poner en `hibernate-configuration` para nosotros es el `persistence.xml`

Pd, recorda que para el TP asegurarte que tu ayudante va a poder levantar el código, eso significa que te asegures que va a poder usar un sql server

**Reply by gastonprieto** [reply]
te sirvio @23737k ?

**Reply by 23737k** [reply]
No, al final lo hice con postgresql y anduvo. ¡Gracias!
